api/src/mailbox.rs

```diff
@@ -1971,6 +2032,291 @@ pub struct GetImageInfoResp {
 }
 impl Response for GetImageInfoResp {}
 
+// CM_AES_CBC_ENCRYPT_INIT
+#[repr(C)]
+#[derive(Debug, IntoBytes, FromBytes, KnownLayout, Immutable, PartialEq, Eq)]
+pub struct CmAesCbcEncryptInitReq {
```

Comment:
For GCM is makes sense to have separate commands.

But for all the unauthenticated cipher modes, should we just have a single set of commands that accepts a mode enum?

I think inevitibly someone will ask for another mode since the hw supports it. It would be good to not need to add a bunch of new commands for that.

Reply:
I thought about this and even did that at first, but realized that the encryption and decryption are different enough to warrant a separate command. To follow NIST recommendations, the encryption operation should return the initial random IV that it generates on the first command.

We could just always return that as a parameter and have it be 0 for decryption, but that felt a little weird to me for some reason.

Reply:
We discussed this offline and are going to go with four commands that have a mode so that we can add additional modes later.